sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2922,18 +2922,29 @@ class Dataset[T] private[sql](
    */
   @throws[AnalysisException]
   def withColumnsRenamed(colsMap: Map[String, String]): DataFrame = withOrigin {
```

Comment:
Since we touch `Dataset.scala`, could you include `[SQL]` into the PR title?

Reply:
sounds good